init.lua

```diff
@@ -283,7 +283,7 @@ minetest.register_chatcommand("rank", {
 		if #param == 1 and param[1] == "list" then
 			return true, "Available Ranks: "..ranks.list_plaintext()
 		elseif #param == 2 then
-			if minetest.get_player_by_name(param[1]) then
+			if ranks.get_rank(param[1]) then
```

Comment:
I've just realized this check is probably not needed at all. I'm pretty sure it will always return false for players that don't have a rank yet, so it would be impossible to set the rank of a player with no rank. So yeah, it looks to me as though the entire check could probably be removed.

Reply:
Fixed